cmd/export_ledger_entry_changes.go

```diff
@@ -96,15 +91,20 @@ be exported.`,
 					"trustlines":         {},
 					"liquidity_pools":    {},
 				}
+
 				for entryType, changes := range batch.Changes {
+					if err != nil {
+						cmdLogger.Fatal("could not read ledger close meta: ", err)
```

Comment:
This looks like a copy and paste error. Shouldn't have this error anymore

Reply:
Sorry, i made the changes